src/reducers/projects.reducer.js

```diff
@@ -32,7 +32,7 @@ export const initialState = {
   selectedId: null,
 };
 
-const byId = (state: ById = initialState.byId, action: Action) => {
+const byIdReducer = (state: ById = initialState.byId, action: Action) => {
```

Comment:
I think this change makes sense, probably best to call out reducers as such, since there are other things in this file.